### PR TITLE
CHANGELOG: update

### PR DESCRIPTION
**Pull request by hasbro17**
[skip ci]

/cc @hongchaodeng

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Changed etcd pod member names to be unique by having a random suffix instead of a sequence number e.g `example-etcd-cluster-2885z`
```
Comment (hongchaodeng) [inline CHANGELOG.md]: Compare what it looked like before and looks now.
Also mention about compatibility.

**Comment by hongchaodeng** [comment]
Also need to mention that we fixed a bug that restore might fail.

**Comment by hongchaodeng** [comment]
It's mystic convention. But object name could not exceed 64 characters:
https://github.com/kubernetes/apimachinery/blob/1168e538ea3ccf444854d1fdd5681d2d876680a7/pkg/util/validation/validation.go#L33

We also need to subtract the five characters from it.

**Comment by fanminshi** [comment]
lgtm

**Pull request merged**